### PR TITLE
Increase e2e timeout to 6 hours and reduce the run frequency to every…

### PR DESCRIPTION
**Pull request by chris-hinds**
… 6 hours instead of 3 hours

- Increases the e2e timeout to 6 hours
- Changes the run frequency to every 6 hours instead of every 3 hours

### CODE REVIEW

File: Jenkinsfile-e2e [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env groovy
 library 'Simorgh'
 
-// Run latest every 3 hours
-def cron_string = env.BRANCH_NAME == "latest" ? "0 0,6,9,12,15,18,21 * * *" : ""
+// Run latest every 6 hours
+def cron_string = env.BRANCH_NAME == "latest" ? "0 */6 * * *" : ""
```
Comment (andrewscfc) [inline Jenkinsfile-e2e:5]: Looks like it didn't used to run exactly every 3 hours, taking a little break before midnight and in the early hours! I assume there wasn't a reason for this?

**Review by andrewscfc — APPROVED** [review]

File: Jenkinsfile-e2e (hunk 1 above)
Comment (chinds185a) [reply Jenkinsfile-e2e]: We used to have issues with the tests running at 3am so it was disabled for 3am.

**Review by sareh — APPROVED** [review]

File: Jenkinsfile-e2e (hunk 1 above)
Comment (chris-hinds) [reply Jenkinsfile-e2e:5]: We previously had to skip runs at 3am as they would never work

**Review by jamesdonoh — APPROVED** [review]
Regrettably, while we look for a longer-term solution, 👍

**Pull request merged**